dash/frontend/src/app/core/dialogues/show-json-data/show-json-data.component.ts

```diff
@@ -27,7 +27,7 @@ export class ShowJsonDataComponent implements OnInit {
   displayedColumns = ['calendarDate', 'namespace', 'pod', 'image', 'message'];
   namespace = this.data.content.namespace;
   date = this.data.content.calendarDate;
-  dateTime  = new Date(this.date);
+  dateTime  = new Date(+(this.data.content.timestamp));
```

Comment:
Is this guaranteed to be populated? If not, we may want to handle cases where it is not populated elegantly.

Reply:
Well, I'm sure content won't be null, but I wonder if timestamp might ever be null. If it were null, what would the output of the code do?

Reply:
IE - would this code potentially crash the application? I'm not sure, but you should check since its operating on that field: 

    dateTime  = new Date(+(null));